privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -0,0 +1,1195 @@
+        :type ret: user defined
+        :return: subsection if key exists or user defined
+        :rtype: dict or scalar
+        """
+        res = {
+            'type': 'webauthn',
+            'title': 'WebAuthn Token',
+            'description': 'WebAuthn: Enroll a Web Authentication token.',
+            'init': {},
+            'config': {},
+            'user': ['enroll'],
+            # This tokentype is enrollable in the UI for...
+            'ui_enroll': ["admin", "user"],
+            'policy': {
+                SCOPE.AUTH: {
+                    WEBAUTHNACTION.ALLOWED_TRANSPORTS: {
+                        'type': 'str',
+                        'desc': _("A list of transports to prefer to communicate with WebAuthn tokens."
+                                  "Default: usb ble nfc internal lightning (All standard transports)")
+                    },
+                    WEBAUTHNACTION.TIMEOUT: {
+                        'type': 'int',
+                        'desc': _("The time in seconds the user has to confirm authorization on his WebAuthn token. " 
+                                  "Note: You will want to increase the ChallengeValidityTime along with this. "
+                                  "Default: 60")
+                    },
+                    WEBAUTHNACTION.USER_VERIFICATION_REQUIREMENT: {
+                        'type': 'str',
+                        'desc': _("Whether the user's identity should be verified when authenticating with a WebAuthn "
+                                  "token. Default: preferred (verify the user if supported by the token)"),
+                        'value': [
+                            'required',
+                            'preferred',
+                            'discouraged'
+                        ]
+                    },
+                    ACTION.CHALLENGETEXT: {
+                        'type': 'str',
+                        'desc': _('Use an alternate challenge text for telling the '
+                                  'user to confirm with his WebAuthn device')
+                    }
+                },
+                SCOPE.AUTHZ: {
+                    WEBAUTHNACTION.AUTHENTICATOR_SELECTION_LIST: {
+                        'type': 'str',
+                        'desc': _("A list of WebAuthn authenticators acceptable for authorization, given as"
+                                  "a space-separated list of AAGUIDs. Per default all authenticators are acceptable.")
+                    },
+                    WEBAUTHNACTION.REQ: {
+                        'type': 'str',
+                        'desc': _("Only the specified WebAuthn-tokens are authorized.")
+                    }
+                },
+                SCOPE.ENROLL: {
+                    WEBAUTHNACTION.RELYING_PARTY_NAME: {
+                        'type': 'str',
+                        'desc': _("A human readable name for the organization rolling out WebAuthn tokens."),
```

Comment:
You can add a new group like:

    "group": "WebAuthn"

which will create a new accordeon in the UI.